operator/controllers/config/app_info.go

```diff
@@ -6,6 +6,8 @@ type AppInfo struct {
 	Name                 string
 	Port                 int32
 	Image                string
+	MinReplicas          int32
```

Comment:
@khaosdoctor can we leave these out of the app config and use `httpso.Spec.MinReplicas`/`MaxReplicas` instead?

Reply:
Also resolved in last commit